app/containers/FileTree.js

```diff
@@ -35,7 +35,7 @@ function buildTree(folderName, files, handleClick, forceOpen = false) {
         return <File key={value} name={filename} path={value} handleClick={handleClick} />;
       }
       return <UnsupportedFile key={value} name={filename} />;
-    }).valueSeq();
+    }).valueSeq().toArray();
```

Comment:
Is this necessary? I would have thought the iterable produced by `.valueSeq()` would be viable for component children.